lib/line/bot/v2/webhook_parser.rb

```diff
@@ -11,8 +11,18 @@ module V2
       class WebhookParser
         class InvalidSignatureError < StandardError; end
 
-        def initialize(channel_secret:)
+        # Initialize webhook parser
+        #
+        # @param channel_secret [String]
+        #   The channel secret used for signature verification.
+        # @param skip_signature_verification [Proc, nil]
+        #   A function that determines whether to skip webhook signature verification.
+        #   If the function returns true, the signature verification step is skipped.
+        #   This can be useful in scenarios such as when you're in the process of updating
+        #   the channel secret and need to temporarily bypass verification to avoid disruptions.
```

Comment:
Probably it's more explicit if this document explains "the function must returns boolean value", though this already says "If the function returns true"?